client/client.go

```diff
@@ -282,3 +282,7 @@ func (c *Client) CreateTenantUsingMspPortal(ctx context.Context, createInput ten
 func (c *Client) ReadMspManagedTenant(ctx context.Context, readByUidInput tenants.ReadByUidInput) (*tenants.MspTenantOutput, error) {
```

Comment:
Maybe change it to `ReadMspManagedTenantByUid` to clarify the difference?

Reply:
Done